<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Capex_MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>781</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCapex">
    <property name="title">
     <string>Capex</string>
    </property>
    <addaction name="actionCapex_2"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionLocation"/>
    <addaction name="actionUnit"/>
    <addaction name="actionVendor"/>
    <addaction name="actionUom"/>
    <addaction name="actionSubCategory"/>
    <addaction name="actionStatus"/>
    <addaction name="actionRecommendation"/>
    <addaction name="actionOrigin"/>
    <addaction name="actionNature"/>
    <addaction name="actionFrequency"/>
    <addaction name="actionExpType"/>
    <addaction name="actionCurrency"/>
    <addaction name="actionArea"/>
    <addaction name="actionApprover"/>
    <addaction name="actionCategory"/>
   </widget>
   <addaction name="menuCapex"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCapex"/>
   <addaction name="actionOptions"/>
  </widget>
  <action name="actionCapex">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/Development.svg</normaloff>:/icons/icons/Development.svg</iconset>
   </property>
   <property name="text">
    <string>Capex</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionCapex_2">
   <property name="text">
    <string>Capex</string>
   </property>
   <property name="toolTip">
    <string>Capex</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="icon">
    <iconset>
     <normaloff>icons/Support.svg</normaloff>
     <normalon>:/icons/icons/Support.svg</normalon>icons/Support.svg</iconset>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionUnit">
   <property name="text">
    <string>Unit</string>
   </property>
   <property name="toolTip">
    <string>Unit</string>
   </property>
  </action>
  <action name="actionLocation">
   <property name="text">
    <string>Location</string>
   </property>
   <property name="toolTip">
    <string>Location</string>
   </property>
  </action>
  <action name="actionVendor">
   <property name="text">
    <string>Vendor</string>
   </property>
  </action>
  <action name="actionUom">
   <property name="text">
    <string>Uom</string>
   </property>
  </action>
  <action name="actionSubCategory">
   <property name="text">
    <string>SubCategory</string>
   </property>
  </action>
  <action name="actionStatus">
   <property name="text">
    <string>Status</string>
   </property>
  </action>
  <action name="actionRecommendation">
   <property name="text">
    <string>Recommendation</string>
   </property>
  </action>
  <action name="actionOrigin">
   <property name="text">
    <string>Origin</string>
   </property>
  </action>
  <action name="actionNature">
   <property name="text">
    <string>Nature</string>
   </property>
  </action>
  <action name="actionFrequency">
   <property name="text">
    <string>Frequency</string>
   </property>
  </action>
  <action name="actionExpType">
   <property name="text">
    <string>ExpType</string>
   </property>
  </action>
  <action name="actionCurrency">
   <property name="text">
    <string>Currency</string>
   </property>
  </action>
  <action name="actionArea">
   <property name="text">
    <string>Area</string>
   </property>
  </action>
  <action name="actionApprover">
   <property name="text">
    <string>Approver</string>
   </property>
  </action>
  <action name="actionCategory">
   <property name="text">
    <string>Category</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
